src/components/ComponentRelation/__tests__/utils.spec.ts

```diff
@@ -1,5 +1,71 @@
 import { ComponentRelationNudgeType } from '../type';
-import { componentRelationValidationSchema, transformNudgeData } from '../utils';
+import {
+  componentRelationValidationSchema,
+  computeNudgeDataChanges,
+  transformNudgeData,
+} from '../utils';
+
+describe('transformNudgeData', () => {
```

Comment:
unit tests for `transformNudgeData` already exist, check below Line 70. Can we remove these? or Are these different?

Reply:
It's a dupe, removed it.